configure.ac

```diff
@@ -3031,6 +3031,31 @@ if test X"$want_h2" != Xno; then
 fi
 
 dnl **********************************************************************
+dnl Check for zsh completion path
+dnl **********************************************************************
+
+OPT_ZSH_FPATH=default
+AC_ARG_WITH(zsh-functions-dir,
+AC_HELP_STRING([--with-zsh-functions-dir=PATH],[Install zsh completions to PATH])
+AC_HELP_STRING([--without-nghttp2],[Do not install zsh completions]),
```

Comment:
I tested the `--with`-with-argument and `--with`-without-argument codepaths but not the `--without-` codepath. Fixed.